modules/op-scoping-github-token.adoc

```diff
@@ -0,0 +1,103 @@
++
+[source,yaml]
+----
+  spec:
+      platforms:
+        openshift:
+          pipelinesAsCode:
+            enable: true
+            settings:
+              secret-github-app-token-scoped: false
+              secret-github-app-scope-extra-repos: "owner2/project2, owner3/project3"
+----
++
+. To set repository level configuration for scoping the GitHub token, specify the additional repositories the `github_app_token_scope_repos` key, as in the following example:
++
+[source,yaml]
+----
+  apiVersion: "pipelinesascode.tekton.dev/v1alpha1"
+  kind: Repository
+  metadata:
+    name: test
+    namespace: test-repo
+  spec:
+    url: "https://github.com/linda/project"
+    settings:
+      github_app_token_scope_repos:
+      - "owner/project"
+      - "owner1/project1"
+----
++
+In this example, the `Repository` custom resource is associated with the `linda/project` repository in the `test-repo` namespace. The scope of the generated GitHub token is extended to the `owner/project` and `owner1/project1` repositories, as well as the `linda/project` repository. These repositories must exist under the `test-repo` namespace.
++
+[NOTE]
+====
+The additional repositories can be public or private, but must reside in the same namespace as the repository with which the `Repository` resource is associated.
+
+If any of the repositories does not exist in the namespace, the scoping of the GitHub token fails with an error message as in the following example:
+
+[,console]
+----
+failed to scope GitHub token as repo owner1/project1 does not exist in namespace test-repo
+----
+====
+
+.Result
+
+The generated GitHub token enables access to additional repositories that you configured in the global and repository level configuration, as well as the original repository from which the {pac} payload files come.
+
+If you provide both global configuration and repository level configuration, the token is scoped to all the repositories from both configurations, as in the following example.
+
+.`TektonConfig` custom resource
+[source,yaml]
+----
+  spec:
+      platforms:
+        openshift:
+          pipelinesAsCode:
+            enable: true
+            settings:
+              secret-github-app-token-scoped: false
+              secret-github-app-scope-extra-repos: "owner2/project2, owner3/project3"
+----
+
+.`Repository` custom resource
+[source,yaml]
+----
+ apiVersion: "pipelinesascode.tekton.dev/v1alpha1"
+ kind: Repository
+ metadata:
+   name: test
+   namespace: test-repo
+ spec:
+   url: "https://github.com/linda/project"
+   settings:
+     github_app_token_scope_repos:
+     - "owner/project"
+     - "owner1/project1"
+----
+
+The GitHub token is scoped to the following repositories: `owner/project`, `owner1/project1`, `owner2/project2`, `owner3/project3`, `linda/project`.
```

Comment:
I don't think that we generally have results sections, and it feels like this information about configuring both could be moved instead to under the bullet list where it's explained that you can configure global or per repository access? I feel like this info would be more useful to users at the beginning rather than here at the end.

Reply:
The problem is that I need to provide examples, and in the beginning examples would not work as the syntax is not explained yet. I honestly see no better solution than a results section for this, unless maybe a note is better?